setup.py

```diff
@@ -38,7 +38,8 @@
         "tensorflow;platform_machine!='arm64'",
```

Comment:
@florian-huber Since I added a script for calculating the MS2Deepscore scores rdkit is now also required. I think a good approach could be to separate the running of MS2Deepscore from the training dependencies. What do you think.

Reply:
In addition we could also split the tests into two separate test types, to make sure the running pipelines work without the need of the training dependencies.